test/helpers/wrappers.go

```diff
@@ -32,6 +32,7 @@ const (
 // Ping returns the string representing the ping command to ping the specified
 // endpoint.
 func Ping(endpoint string) string {
+	// TODO: add W -2 ?
```

Comment:
Maybe `W -2` for Pingfail and `W -5` for this case?

Reply:
I can do this as a follow-up PR? The current functionality works OK, this was more of a future thing to look into.